Kite SDK/PSPrintSDK/KitePrintSDK-Info.plist

```diff
@@ -48,8 +48,6 @@
 	<key>UISupportedInterfaceOrientations</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
-		<string>UIInterfaceOrientationLandscapeLeft</string>
-		<string>UIInterfaceOrientationLandscapeRight</string>
```

Comment:
Ah my bad… This means a few screens rotate if you apply filters and I skip the OLProductHomeViewController, need to look into that… 

> On 9 Apr 2015, at 15:33, Konstadinos Karayannis <notifications@github.com> wrote:
> 
> In Kite SDK/PSPrintSDK/KitePrintSDK-Info.plist <https://github.com/OceanLabs/iOS-Print-SDK/pull/72#discussion_r28066570>:
> 
> > @@ -48,8 +48,6 @@
> >  	<key>UISupportedInterfaceOrientations</key>
> >  	<array>
> >  		<string>UIInterfaceOrientationPortrait</string>
> > -		<string>UIInterfaceOrientationLandscapeLeft</string>
> > -		<string>UIInterfaceOrientationLandscapeRight</string>
> I added rotation to the demo app so that we can test that none of our screens rotate even if the host app does. Remember how the credit card screen rotated? We probably would have caught that if the demo app could rotate (it didn't at that time)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/OceanLabs/iOS-Print-SDK/pull/72/files#r28066570>.
>